src/util/JoinAlgorithms/JoinAlgorithms.h

```diff
@@ -1228,7 +1253,8 @@ struct BlockZipperJoinImpl {
   // `side.undefBlocks_` and skipped for subsequent processing. The first block
   // containing defined values is split and the defined part is stored in
   // `side.currentBlocks_`.
-  void findFirstBlockWithoutUndef(auto& side) {
+  void findFirstBlockWithoutUndef(
+      ad_utility::SameAsAny<LeftSide, RightSide> auto& side) {
```

Comment:
The only concepts you are using are `SameAsAny<LeftSide, RightSide>` and `SameAsAny<CurrentBlocksLeft, CurrentBlocksRight>`.
Can we explore if we can remove some of the redundancy?
My first approach is: With concepts it doesn't work, they can't be local, you cant shorten their names, etc.

So a syntax could be to have a `static constexpr bool` and then write
```
template <typename T> requires LeftOrRightSide<T>
```
But that is not at all shorter....


If you want to....
Bad word Bad word Bad word...


** MACRO **

(don't forget to Undef it).

But maybe this all isn't worth it.